### ISSUE TITLE
Invalid paths in umbrella warnings

### ISSUE THREAD

**Issue by lukaszsamson** [esl/gradient#143]
When run in the root of umbrella wrong paths are returned

Expected:
```
apps/my_app/lib/my_app/my_module.ex: The variable my_var on line 50 is expected to have type pos_integer() but it has type integer()
```
Is:
```
apps/my_app/apps/my_app/lib/my_app/my_module.ex: The variable my_var on line 50 is expected to have type pos_integer() but it has type integer()
```

**Comment by erszcz** [comment]
I think that's solved in #124. We just talked with @luk-pau-es and he's kind enough to craft a more specific PR to handle just this.

**Comment by erszcz** [comment]
Fixed with #146.
